src/Microsoft.Azure.WebJobs.Host/Filters/FunctionExecutedContext.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
```

Comment:
I moved all the Filters stuff under a new Host/Filters directory. That's why many of these show as additions instead of modifications.